package/kernel/mwlwifi/Makefile

```diff
@@ -27,8 +27,9 @@ include $(INCLUDE_DIR)/package.mk
 
 define KernelPackage/mwlwifi
   SUBMENU:=Wireless Drivers
-  TITLE:=Marvell 88W8864 wireless driver
+  TITLE:=Marvell 88W8864/88W8897/88W8964 wireless driver
   DEPENDS:=+kmod-mac80211 +@DRIVER_11N_SUPPORT +@DRIVER_11AC_SUPPORT +@DRIVER_11W_SUPPORT @PCI_SUPPORT @TARGET_mvebu
+  DEPENDS+=@(PACKAGE_mwlwifi-firmware-88w8864||PACKAGE_mwlwifi-firmware-88w8897||PACKAGE_mwlwifi-firmware-88w8964)
```

Comment:
Can you please make the dependency the other way around.
Add a "DEPENDS:=+kmod-mwlwifi" to all the FW packages.